Mostly adding suffixes.

Last two hunks: patch to work with the older version of texinfo that we
have in base (we don't want automake to depend on too many other ports..);
the @subsection/@node one seems to be a bug in upstream's file which has
since been fixed, the other is @geq which isn't supported in older texinfo.

Index: doc/automake.texi
--- doc/automake.texi.orig
+++ doc/automake.texi
@@ -1,7 +1,8 @@
 \input texinfo   @c -*-texinfo-*-
 @c %**start of header
-@setfilename automake.info
-@settitle automake
+@setfilename automake${SUFFIX}.info
+@dircategory Programming & development tools
+@settitle automake${SUFFIX}
 @documentencoding UTF-8
 @documentlanguage en
 @setchapternewpage off
@@ -40,13 +41,13 @@ section entitled ``GNU Free Documentation License.''
 
 @dircategory Software development
 @direntry
-* Automake: (automake).         Making GNU standards-compliant Makefiles.
+* automake${SUFFIX}: (automake${SUFFIX}).         Making GNU standards-compliant Makefiles.
 @end direntry
 
 @dircategory Individual utilities
 @direntry
-* aclocal-invocation: (automake)aclocal Invocation.   Generating aclocal.m4.
-* automake-invocation: (automake)automake Invocation. Generating Makefile.in.
+* aclocal${SUFFIX}: (automake${SUFFIX})aclocal Invocation.   Generating aclocal.m4.
+* automake${SUFFIX}: (automake${SUFFIX})automake Invocation. Generating Makefile.in.
 @end direntry
 
 @titlepage
@@ -6398,8 +6399,8 @@ fixed, and a parallel make could invoke more than one 
 * Linking Multiple Yacc Parsers::
 @end menu
 
-@subsection Linking Multiple Yacc Parsers
 @node Linking Multiple Yacc Parsers
+@subsection Linking Multiple Yacc Parsers
 
 @cindex Linking multiple @command{yacc} parsers
 
@@ -7865,7 +7866,7 @@ AM_PATH_PYTHON([2.5])
 
 @noindent
 This is fine when Python is an absolute requirement for the package.
-If Python @geq{} 2.5 was only @emph{optional} for the package,
+If Python >= 2.5 was only @emph{optional} for the package,
 @code{AM_PATH_PYTHON} could be called as follows.
 
 @example
